osbuild-composer.spec

```diff
@@ -148,6 +149,7 @@ install -m 0755 -vp _bin/osbuild-tests                          %{buildroot}%{_l
 install -m 0755 -vp _bin/osbuild-weldr-tests                    %{buildroot}%{_libexecdir}/tests/osbuild-composer/
 install -m 0755 -vp _bin/osbuild-dnf-json-tests                 %{buildroot}%{_libexecdir}/tests/osbuild-composer/
 install -m 0755 -vp _bin/osbuild-image-tests                    %{buildroot}%{_libexecdir}/tests/osbuild-composer/
+install -m 0755 -vp _bin/cloud-cleaner                      %{buildroot}%{_libexecdir}/tests/osbuild-composer/
```

Comment:
There's a whitespace issue.

Reply:
Fixed :)

Reply:
Hmm, I gave this another look and I think we should put the cloud-cleaner into `%{buildroot}%{_libexecdir}/osbuild-composer/`. I think in the future we want to be able to just iterate through the test libexec directory and run them all so we cannot put the cloud-cleaner there. Sorry!

Reply:
That is correct, but it also seems that the cloud cleaner cannot access any environment variables. Otherwise I cannot explain why the credentials structure is `nil`. Though I'm not sure why the environment variables are not available. I'm trying to read more about the "post" section, but it seems they should be available even in there.